src/components/modalWindows/deleteMovie/deleteMovie.js

```diff
@@ -38,6 +40,12 @@ const DeleteMovie = (props) => {
   )
 }
 
+const mapDispatchToProps = (dispatch) => {
+  return {
+    deleteMovie: (id) => {dispatch(removeMovie(id))},
```

Comment:
You can use short form `deleteMovie: removeMovie`

Reply:
done